repos/system_upgrade/common/actors/removeleftoverpackages/libraries/removeleftoverpackages.py

```diff
@@ -0,0 +1,56 @@
+from leapp.libraries.common.rhsm import skip_rhsm
+from leapp.libraries.common.rpms import get_installed_rpms
+from leapp.libraries.stdlib import api, CalledProcessError, run
+from leapp.models import LeftoverPackages, RemovedPackages, RPM
+
+
+def get_leftover_packages():
```

Comment:
One more thing, functions private to the library should begin with `_`.

Reply:
Added the `_`.